src/app/app.component.ts

```diff
@@ -41,4 +44,22 @@ export class AppComponent {
 
     });
   }
+
+  ngOnInit(): void {
+    const appTitle = this.titleService.getTitle();
+    this.router
+      .events.pipe(
+        filter(event => event instanceof NavigationEnd),
+        map(() => {
+          const child = this.activatedRoute.firstChild;
+          if (child.snapshot.data.title) {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+            return child.snapshot.data.title;
+          }
+          return appTitle;
+        })
+      ).subscribe((ttl: string) => {
+        this.titleService.setTitle(ttl);
```

Comment:
The issue has been updated (you might not have seen it yet) to say that the page-titles need a prefix of "Micronutrient Action Policy Support (MAPS):".